pyquil/quilbase.py

```diff
@@ -144,7 +144,7 @@ def dagger(self):
         return self
 
     def __repr__(self):
-        return "<" + "".join([mod + " " for mod in self.modifiers]) + "Gate " + str(self) + ">"
+        return "<Gate " + str(self) + ">"
```

Comment:
Again out of scope for this PR - but our `repr` method should just be equal to `str`

Reply:
I can dig it